runhouse/resources/hardware/cluster.py

```diff
@@ -141,6 +142,10 @@ def address(self, addr):
         self.ips = self.ips or [None]
         self.ips[0] = addr
 
+    @property
+    def internal_ips(self):
+        return self.ips
```

Comment:
self.ips refers to external ips right?

Reply:
![image](https://github.com/user-attachments/assets/72f9c46b-5164-44b9-8e7c-e203f9e393a7)
Whatever Donny has done here works, but yes

Reply:
🤯

Reply:
No it's because for a static cluster internal_ips and external_ips are the same, and we need the cluster to have an internal_ips property just like on_demand_cluster, or the dask (and ray) start sequence for each needs to be different (we need to use an internal ip to connect from the worker nodes to the head node).

Reply:
🤯🤯🤯